Gems/ROS2/Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -120,6 +120,25 @@ namespace ROS2
             return;
         }
 
+        auto& spawnable = m_controller.GetSpawnables().find(spawnableName)->second;
+        if (spawnable->IsLoading())
+        {
+            // This check is done during the simulation. All assets in the build of the simulation are loaded and processed.
```

Comment:
This is Editor-only situation, correct? Please rephrase the comment for clarity.